salt/state.py

```diff
@@ -1994,6 +2087,10 @@ def check_requisite(self, low, running, chunks, pre=False):
                 if tag not in run_dict:
                     fun_stats.add('unmet')
                     continue
+                if run_dict[tag].get('proc'):
+                    # Run in parallel, first wait for a touch and then recheck
+                    time.sleep(0.2)
```

Comment:
Again, this magic value seems like a bit too high, especially if this parallel task is going to block other dependent tasks.